source/collections/time-series-collection.txt

```diff
@@ -15,6 +15,18 @@ Create a Time Series Collection
 :manual:`Time series collections </core/timeseries-collections/>` efficiently 
 store sequences of measurements over a period of time. 
 
+Limitations
+-----------
+
+The following restrictions and limitations apply when creating a time series 
+collection:
+
+- :guilabel:`Custom collation` is the only :guilabel:`Advanced Collection Option` 
+  that can be used alongside your time series collation.  
```

Comment:
```suggestion
  that is usable with a time series collection.  
```
Was "collation" intended, or should this be "collection" as with the suggestion above?

Reply:
Looks like this was a typo that we didn't catch when this content was first written! Fixed